user/languages/javascript-with-nodejs.md

```diff
@@ -164,6 +164,7 @@ Note that `yarn` requires Node.js version 4 or later.
 If the job does not meet this requirement, `npm install` is used
 instead.
 
+If Yarn is supported, test command is run using `yarn test` instead of `npm test`.
```

Comment:
I think the best place to put this information is the "Default Build Script" section above. But very much like here, adding a "Yarn is supported" subsection.

Reply:
Is that better